### ISSUE TITLE
Compare selected tab context menu shortcuts

### ISSUE THREAD

**Issue by XeonG** [in4margaret/compareit#2]
If you right click on VScode document tabs, you don't have any context menu shortcut for 'select for compare' and 'compare with selected'

Having to use the sidebar menu is a hassle
1. Sometimes just getting the context menu to show up with the select for compare shortcuts is annoying.
2. Sometimes you have the tab documents you want to compare already opened, however the sidebar is closed, or maybe the sidebar is opened up on something else like the extensions sidebar.

**Comment by JoanEliot** [comment]
Agree. Would be a big help.

**Comment by XeonG** [comment]
yeah and the most stupid thing is you can't even copy paste text into 2 new tabs and compare those... they have to be saved documents.. its sad really good be really good but the entire implementation leading up the actual 2tab comparison is well ..naff


and well even after you have the comparison its ,bleh still not great as you can't edit either document anyway
